code/modules/language/language_holder.dm

```diff
@@ -232,6 +232,17 @@ Key procs
 	spoken_languages = list(/datum/language/xenocommon = list(LANGUAGE_ATOM))
 	blocked_languages = list(/datum/language/common = list(LANGUAGE_ATOM))
 
+//skyrat change start
+/datum/language_holder/terror_spider
+	understood_languages = list(/datum/language/terrorspider = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/terrorspider = list(LANGUAGE_ATOM))
+
+/datum/language_holder/terror_spider_high
+	understood_languages = list(/datum/language/terrorspider = list(LANGUAGE_ATOM),
+								/datum/language/common = list(LANGUAGE_ATOM))
+	spoken_languages = list(/datum/language/terrorspider = list(LANGUAGE_ATOM))
+
+//skyrat change stop
```

Comment:
This likewise, can go into a separate file, rather than cramming it into core files.